src/common/models/Projection.ts

```diff
@@ -397,6 +404,15 @@ export class Projection {
       return null;
     }
 
+    const peopleVaccinated = Projection.getActualOrDefault(
```

Comment:
I don't think you actually need this helper function!  pretty sure you can just do: 
`const peopleVaccinated = actuals.vaccinationsCompleted ?? (percentVaccinated / 100.0) * population`

Reply:
Done in 6973ce2